misc/scripts/install-local.sh

```diff
@@ -30,36 +30,36 @@ source "${STGDIR}/scripts/checks.sh" || {
 
 function cleanup() {
     if [[ -n $KEEP ]]; then
-        rm -rf "/tmp/pacstall-keep/$name"
-        mkdir -p "/tmp/pacstall-keep/$name"
+        rm -rf "/tmp/pacstall-keep/$pkgname"
+        mkdir -p "/tmp/pacstall-keep/$pkgname"
         if [[ -f /tmp/pacstall-pacdeps-"$PACKAGE" ]]; then
-            sudo mv /tmp/pacstall-pacdep/* "/tmp/pacstall-keep/$name"
+            sudo mv /tmp/pacstall-pacdep/* "/tmp/pacstall-keep/$pkgname"
         else
-            sudo mv "${SRCDIR:?}"/* "/tmp/pacstall-keep/$name"
+            sudo mv "${SRCDIR:?}"/* "/tmp/pacstall-keep/$pkgname"
         fi
     fi
     if [[ -f "/tmp/pacstall-pacdeps-$PACKAGE" ]]; then
         sudo rm -rf "/tmp/pacstall-pacdeps-$PACKAGE"
         sudo rm -rf /tmp/pacstall-pacdep
     else
         sudo rm -rf "${SRCDIR:?}"/*
-        # just in case we quit before $name is declared, we should be able to remove a fake directory so it doesn't exit out the script
-        sudo rm -rf "${STOWDIR:-/usr/src/pacstall}/${name:-raaaaaaaandom}"
+        # just in case we quit before $pkgname is declared, we should be able to remove a fake directory so it doesn't exit out the script
+        sudo rm -rf "${STOWDIR:-/usr/src/pacstall}/${pkgname:-raaaaaaaandom}"
```

Comment:
Can this actually be random and not fake text that says random

Reply:
or maybe just like `pacstall-MMDDYYYY-Thh-mm-ss`

Reply:
wait I have an idea.